src/style.scss

```diff
@@ -57,19 +49,19 @@ hr {
 // it changes the width of the container and the width of the other elements around it
 // Switching the container to "fluid" didn't fix it, so this was unfortunately the best way I could
 // find to solve the issue; by fixing the container width for medium screens and up
-@media (min-width: 768px) and (max-width: 991px) {
+@media (min-width: $md) and (max-width: $lg) {
```

Comment:
Don't you run into conflicting breakpoints unless you do `calc($lg - 1)` ?

Reply:
@ayan4m1 just tested it, and it doesn't seem to make a difference but I can change that if you think it would be best practice to do so! 😄